src/components/CandidateAvatar/CandidateAvatar.scss

```diff
@@ -0,0 +1,70 @@
+.CandidateAvatar {
+    position: relative;
+    display: inline-block;
+    height: 72px;
+    width: 72px;
+
+    &__ring {
+        width: 72px;
+        height: 72px;
+    }
+
+    &__circle {
+        fill: none;
+        stroke: var(--color-neutral);
```

Comment:
this is never used. Why do you add it here?

Reply:
Yeah... not really necessary indeed